### PR TITLE
Unbound Links to raise awareness.

### PR DESCRIPTION
**Pull request by Bas-Man**
This is another possible DNS service that could be used.
I am including two links to RedHat articles related to this.

This should not be merged until some discussion is made about if the DNS page should be further broken down into folders.
This will probably cause conflict with pull request #66 and could be merged into that pull request after discussion.

### CODE REVIEW

**Review by IronicBadger — CHANGES_REQUESTED** [review]
Please remove full stops in title frontmatter.

Also, title of internal DNS server is misspelled as sever.

**Comment by IronicBadger** [comment]
Please create a new category in the nav for DNS at the top level, I don't think it needs to live under System Administration.

File: docs/system-administration/DNS/debian-stretch.md [hunk 1]
```diff
@@ -0,0 +1,721 @@
...
+title: Debian (Stretch / Buster) DNS Configuration.
+summary: A guide on how to setup Primary and Secondary caching servers with zone authority under Debian (stretch and Buster).
+date: 2020-11-26
+authors:
+  - Adam Spann
+---
+
+## A note of warning!
+  This guide is only intended for those that plan to use these servers within their own networks running behind a NAT router. Or with the service only available on the internal interfaces. This configure is not intended for WAN (Wide Area Network) interfaces.
+
```
Comment (IronicBadger) [inline docs/system-administration/DNS/debian-stretch.md:11]: Needs an exposition - what is the purpose of this page and what do you hope to communicate with it?

File: docs/system-administration/DNS/unbound-dns.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Unbound DNS.
+summary:
+date: 2020-12-15
+authors:
+  - Adam Spann
+---
+# Unbound DNS. A Bind Alternative for RedHat, CentOS and Fedora.
```
Comment (IronicBadger) [inline docs/system-administration/DNS/unbound-dns.md:8]: Could use a little more padding perhaps?

Comment (Bas-Man) [reply docs/system-administration/DNS/unbound-dns.md:8]: Padding added. 

Though I think this pull request can be closed per comments on pull-request #66

File: docs/system-administration/DNS/debian-stretch.md (hunk 1 above)
Comment (Bas-Man) [reply docs/system-administration/DNS/debian-stretch.md:11]: I believe this has been done in my latest update.

**Comment by Bas-Man** [comment]
This pull request will be closed once I the full access.
Unbound article was merged into #66

**Comment by Bas-Man** [comment]
This was merged in to pull-request #66

**Pull request closed without merging**